app/dashboard/static/js/app/view-tests-all.2018.9.js

```diff
@@ -283,45 +163,56 @@ require([
         } else {
             columns = [
                 {
-                    data: 'board',
-                    title: 'Board',
+                    data: 'job',
+                    title: 'Tree',
                     type: 'string',
-                    render: ttest.renderBoard
+                    className: 'tree-column',
+                    render: _renderTree
                 },
                 {
-                    data: 'board',
-                    title: 'Total Test Groups',
-                    type: 'num',
-                    searchable: false,
-                    className: 'pull-center',
-                    render: _renderGroupsCount
+                    data: 'git_branch',
+                    title: 'Branch',
+                    type: 'string',
+                    className: 'branch-column',
+                    render: _renderBranch
                 },
                 {
-                    data: 'board',
-                    title: 'Total Unique Labs',
-                    type: 'num',
-                    searchable: false,
-                    className: 'pull-center',
-                    render: _renderLabsCount
+                    data: 'kernel',
+                    title: 'Kernel',
+                    type: 'string',
+                    className: 'kernel-column',
+                    render: _renderKernel
                 },
                 {
-                    data: 'board',
-                    title: '',
+                    data: 'name',
+                    title: 'Test Plan',
+                    type: 'string',
+                    className: 'plan-column',
+                    render: _renderPlan
+                },
+                {
+                    data: 'created_on',
+                    title: 'Date',
+                    type: 'date',
+                    className: 'date-column',
+                    render: ttest.renderDate
+                },
+                {
+                    data: '_id.$oid',
+                    title: 'Status',
                     type: 'string',
                     searchable: false,
                     orderable: false,
-                    className: 'select-column pull-center',
-                    render: _renderDetails
+                    className: 'plan-center pull-center',
+                    render: _renderStatus
                 }
             ];
 
             gTestsTable
                 .data(response)
                 .columns(columns)
-                .order([0, 'asc'])
+                .order([4, 'desc'])
```

Comment:
There appears to be an issue as rows are sorted by date but not by time.  So branches that have more than one revision in one day will appear in random order.  This is typically the case with the staging branch since there are 3 revisions being tested every day.

The boots view doesn't show this issue: https://staging.kernelci.org/boot/
You can see all the `staging-20200408.1` results at the top, and the `staging-20200408.0` results a few pages down.  Maybe this is because we're aggregating results, whereas the boots view doesn't - I'm not sure.  It's not a big issue but it would be a lot better to be able to sort this in true chronological order.

I think the way this is supposed to work, is that the `gTablesUtils.renderDate()` method will either return the string version if the table is asking for a `display` type of rendering or the full ISO timestamp otherwise.  The `display` type should be used to show a string in the table, and the plain type should be used when ordering the rows.  Maybe that's something to debug and check it's doing the correct thing.

Reply:
I am hacking on this :)

Reply:
OK that seems to be fixed now, thanks.